src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerDstu2IT.java

```diff
@@ -53,5 +55,10 @@ void beforeEach() {
 		String ourServerBase = "http://localhost:" + port + "/fhir/";
 		ourClient = ourCtx.newRestfulGenericClient(ourServerBase);
 		ourClient.registerInterceptor(new LoggingInterceptor(true));
+		// Create an HTTP basic auth interceptor
```

Comment:
Can we ensure consistent use of spaces or tabs for indentation (this file are tabs, elsewhere, the same code is a mixture).

Reply:
@ibacher i have fixed them